### PR TITLE
Create hiring-to-help-folk

### PR DESCRIPTION
**Pull request by harryharrold**
Fixes gap in staffing

## Description

- recruitment

### CODE REVIEW

**Review by Nikomus — COMMENTED** [review]
Needs to be a .md file to be picked up and built into a page

**Comment by harryharrold** [comment]
@Nikomus file type added, published date added.

File: src/data/blog/hiring-to-help-folk.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+amp: null
+author_name: 'Neontribe'
+author_slug: 'neontribe'
+author_image: 'images/2019/03/neonround.png'
+author_cover: null
+author_facebook: null
+author_twitter: null
+---
+
+##WWant to help us help folk who need it most?
```
Comment (Nikomus) [inline src/data/blog/hiring-to-help-folk.md:27]:
```suggestion
##Want to help us help folk who need it most?
```

File: src/data/blog/hiring-to-help-folk.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+We're a software development company working on web user interfaces and the APIs that power them. We've got a national reputation for our work in tech-for-good, and we have more and more work coming our way.
+
+Right now, we’re recruiting to fill a specific role in our team. However, we’re also always on the look out for people who would make our teams stronger, so if this doesn’t suit you please see the information at the end.
+
+##WWorking in software development
```
Comment (Nikomus) [inline src/data/blog/hiring-to-help-folk.md:33]:
```suggestion
##Working in software development
```

File: src/data/blog/hiring-to-help-folk.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+You’ll be joining a team working on an established ISO27001 certified project for a valued client with an existing development process. It’s a great opportunity for growth and enjoyment. Neontribe prides itself on skills development, flexible working, part-time hours and supporting return to work and other initiatives to help you enjoy your work. You’ll get 34 days holiday (including bank holidays and a company break midwinter), and £1,000 a year to spend on training. If you like working in an office, we're based in Norwich and Exeter. If you don't, we've been working remotely for years and were very used to doing so. 
+
+##WWorking mainly in PHP
```
Comment (Nikomus) [inline src/data/blog/hiring-to-help-folk.md:37]:
```suggestion
##Working mainly in PHP
```

File: src/data/blog/hiring-to-help-folk.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+featured: 0
+page: 0
+status: 'published'
+language: 'en_US'
+meta_title: null
+meta_description: null
+author_id: 1
+created_at: '2021-7-09T13:25:43.000Z'
+created_by: 1
+published_at: '2021-7-09T17:25:43.000Z'
```
Comment (Nikomus) [inline src/data/blog/hiring-to-help-folk.md:14]:
```suggestion
published_at: '2021-07-09T17:25:43.000Z'
```

**Comment by Nikomus** [comment]
A few tiny fixes suggested. Here's the direct preview link:
https://neontribe-www-git-harryharrold-recruitment-1-neontribe.vercel.app/blog/hiring-to-help-folk

**Comment by harryharrold** [comment]
Yes, all very sensible. Apologies, in fact. And thank you.

**Review by Nikomus — APPROVED** [review]
Looks good! 🚀

**Review by edpe — APPROVED** [review]

**Pull request merged**